src/analysis/retail/shaman/elemental/modules/talents/Stormkeeper.tsx

```diff
@@ -46,6 +273,239 @@ class Stormkeeper extends Analyzer {
     this.damageDoneByBuffedCasts += event.amount + (event.absorbed || 0);
   }
 
+  _explainTimelineWithDetails(cast: SKCast) {
+    const checklistItem = {
+      performance: cast.timeline.performance,
+      summary: <span>Spell order</span>,
+      details: <span>Spell order: See below</span>,
+      check: 'stormkeeper-timeline',
+      timestamp: cast.event.timestamp,
+    };
+
+    const showPrecastSop = SPELLS_SOP_BUFF_REQUIRED.includes(cast.firstRotationCast.ability.guid);
+    const showPrecastMote = [this.stSpender.id].includes(cast.firstRotationCast.ability.guid);
+
+    const showPrecastAny = showPrecastSop || showPrecastMote;
+
+    const extraDetails = (
+      <div
+        style={{
+          overflowX: 'scroll',
+        }}
+      >
+        {showPrecastAny && (
+          <>
```

Comment:
[Consider] Why do the other elements have `p` wrappers but this one doesn't?  Maybe it's a formatting thing, but it was just wondering if maybe you want it to be a paragraph for consistency.